youtube_dl/extractor/heise.py

```diff
@@ -76,7 +74,9 @@ def _real_extract(self, url):
         if not title or title == "c't":
             title = self._search_regex(
                 r'<div[^>]+class="videoplayerjw"[^>]+data-title="([^"]+)"',
-                webpage, 'title')
+                webpage, 'title', default=None)
+        if not title:
+            title = self._og_search_title(webpage)
```

Comment:
Adding another fallback here does not make much sense since title is not used when delegated to kaltura anyway.